cluster/gce/util.sh

```diff
@@ -3206,6 +3348,18 @@ function kube-down() {
         --quiet \
         "${MASTER_NAME}-ip"
     fi
+
+    if [[ ${GCE_PRIVATE_CLUSTER:-} == "true" ]]; then
```

Comment:
This should be done only if REMAINING_MASTER_COUNT is 0 (similarly to line 3340), right?

Reply:
They both are in if [[ REMAINING_MASTER_COUNT -eq 0 ]] starting in 3340 and ending in 3363? :)

Reply:
pagination is hard in bash :)